mlir/docs/Dialects/OpenMPDialect/_index.md

```diff
@@ -0,0 +1,14 @@
+# 'omp' Dialect
+
+The `omp` dialect is for representing directives, clauses and other definitions
+of the OpenMP programming model. This directive-based programming model, defined
```

Comment:
Do you want to leave a pointer to the OpenMP ARB website and maybe even the spec subpage of the OpenMP ARB website?

Reply:
Thanks for the suggestion, done!